cloudfront/iiif.wellcomecollection.org/terraform/cf_origins.tf

```diff
@@ -54,18 +54,23 @@ locals {
       origin_id   = "dlcs_file"
       domain_name = "dlcs.io"
       origin_path = "/file/wellcome/5"
+    },
+    {
+      origin_id   = "pdf_cover"
+      domain_name = "pdf.wellcomecollection.digirati.io"
```

Comment:
CloudFront gets upset trying to apply the prod configuration saying "not all the origins exist", I think it might be that this origin is not contactable.